### PR TITLE
TM: Update the supplementary exercise to use the training manual data

### PR DESCRIPTION
**Pull request by havatv**
Avoid download from the (removed) dropbox folder.

Goal:

Ticket(s): #

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: docs/training_manual/complete_analysis/analysis_exercise.rst [hunk 1]
```diff
@@ -28,13 +28,12 @@ where you should go to look.
...
-In order to solve this problem, you will have to download data from
-https://www.dropbox.com/s/q5evvkizuunrcs0/more_analysis.zip?dl=0 and place it
-under :file:`exercise_data/more_analysis` folder.
-You will use it to find the candidate area
-that is closest to your house. If you don't live in Cape Town (where this
-problem is based) you can choose any house in the Cape Town region. The
-solution will involve:
+The data for this exercise can be found in the
+:file:`exercise_data/more_analysis` folder.
+You are going to find the candidate area that is closest to your house.
```
Comment (DelazJ) [inline docs/training_manual/complete_analysis/analysis_exercise.rst:33]: Few people among the lecturers live in Cape Town so what about skipping the mention of their house and make it impersonal
```suggestion
You are going to find the candidate area that is closest to a house of your choice.
```
Or we indicate a particular house...
PS: I did not read the file to check how the house selection is done later. May need to be reformulated

File: docs/training_manual/complete_analysis/analysis_exercise.rst [hunk 2]
```diff
@@ -28,13 +28,12 @@ where you should go to look.
...
+If you don't live in Cape Town (where this problem is based) you can choose
```
Comment (DelazJ) [inline docs/training_manual/complete_analysis/analysis_exercise.rst:34]:
```suggestion
```

File: docs/training_manual/complete_analysis/analysis_exercise.rst [hunk 3]
```diff
@@ -28,13 +28,12 @@ where you should go to look.
...
+any house in the Cape Town region.
```
Comment (DelazJ) [inline docs/training_manual/complete_analysis/analysis_exercise.rst:35]:
```suggestion
```

**Comment by havatv** [comment]
> How about making the exercise more international?

That is definitely a good idea.
But the purpose of this PR is to avoid the dropbox reference. So I suggest that updating the exercise is done in a separate PR.

**Pull request merged**